src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -222,6 +223,18 @@ def remove_snapshot(self, snapname):
         snappath = self.snapshot_path(snapname)
         rmsnap(self.fs, snappath)
 
+    def snapshot_info(self, snapname):
+        snappath = self.snapshot_path(snapname)
+        try:
```

Comment:
do you see a need to add other keys in the future? (this can be maintained as a map from key_name to xattr_name which can be iterated over to fetch the values)

Reply:
Done